### PR TITLE
Add config for PHE cancer screening site

### PR DESCRIPTION
**Pull request by edds**
https://www.pivotaltracker.com/story/show/100804304

### CODE REVIEW

**Comment by jennyd** [comment]
Is there a story/ticket for this?

**Comment by edds** [comment]
Sorry, yes, added it to the pull request description.

**Comment by jennyd** [comment]
The significant query params need adding - the ticket requests `folder:id`. Other PHE screening sites [also have `monthye`](https://github.com/alphagov/transition-config/blob/master/data/transition-sites/phe_screen_infectiousdiseases.yml#L8), but the [path which it's used for](https://transition.production.alphagov.co.uk/sites/phe_screen_sct/mappings?utf8=%E2%9C%93&path_contains=monthye) on those sites (`/news.php`) [doesn't exist on this site](http://www.cancerscreening.nhs.uk/news.php), so I presume we don't need that one here.

We should also have a `homepage_title` for this site because the PHE org name isn't appropriate as a description of the old site - see an [example](https://github.com/alphagov/transition-config/blob/master/data/transition-sites/phe_screen_infectiousdiseases.yml#L4) on another site and the [README](https://github.com/alphagov/transition-config/blob/master/README.md#site-configuration) for that field.

**Comment by edds** [comment]
Thanks for the assistance @jennyd. Can you tell this is my first transition config file.

**Comment by jennyd** [comment]
We've previously had to [add `www` domains as aliases](https://github.com/alphagov/transition-config/pull/1108) for the other PHE screening sites. That wasn't mentioned in this ticket, but `cancerscreening.nhs.uk` does resolve and looks like an alias, so we should add it to this site now.

**Comment by edds** [comment]
Updated to also have an alias

File: data/transition-sites/phe_screen_cancer.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+site: phe_screen_cancer
+whitehall_slug: public-health-england
+homepage: https://www.gov.uk/topic/population-screening-programmes
+tna_timestamp: 20150506150512
+host: www.cancerscreening.nhs.uk
+options: --query-string folder:id
+homepage_title: 'NHS Cancer Screening Programmes'
+aliases:
+  - cancerscreening.nhs.uk
```
Comment (jennyd) [inline data/transition-sites/phe_screen_cancer.yml]: We've standardised on no indentation for lists in the YAML in this repo (I have a vague recollection that indentation caused parsing errors at some point). There isn't a check in the build for it (there probably should be).

**Pull request merged**